SpotifyApp/SpotifyApp/SpotifyAlbums.json

```diff
@@ -0,0 +1,1711 @@
+{
```

Comment:
I see what you did here. This was a good attempt but your problem was parsing correctly.